pkg/webui/console/store/middleware/logics/notifications.js

```diff
@@ -76,6 +76,7 @@ const getInboxNotificationsLogic = createRequestLogic({
     return {
       notifications: result.notifications,
       totalCount: result.totalCount,
+      unseenTotalCount: result.notifications.filter(notification => !notification.status).length,
```

Comment:
This cannot work because the notifications are paginated. You can only source the unseen count from the `X-Total-Count` header of the notification response, which is what the SDK passes as `totalCount`.